### DISCUSSION TITLE
CLI v5.8 Autoloader\Loader or Loader\Loader

### DISCUSSION THREAD

**Discussion by milacic** [phalcon/cphalcon#16629] [General]
https://docs.phalcon.io/5.8/application-cli/

If I use Phalcon\Autoload\Loader instead of Phalcon\Loader\Loader everything works fine.

I'm I wrong or is it just my mistake?

Thanks :)

**Comment by niden** [comment]
@milacic Thanks for reporting this. This has been fixed. `Phalcon\Autoloader\Loader` is the correct reference.
